ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/textinput/v1/textinput/textinput.js

```diff
@@ -14,30 +14,6 @@
  ~ limitations under the License.
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
 
-use(function () {
-
-  var clientlibsArr = ['core.forms.components.base.v1.editor'];
-  var labelPath = 'core/fd/components/commons/v1/fieldTemplates/label.html';
-  var shortDescriptionPath = "core/fd/components/commons/v1/fieldTemplates/shortDescription.html";
-  var longDescriptionPath = "core/fd/components/commons/v1/fieldTemplates/longDescription.html";
-  var questionMarkPath = "core/fd/components/commons/v1/fieldTemplates/questionMark.html"
-  var errorMessagePath = "core/fd/components/commons/v1/fieldTemplates/errorMessage.html";
-
-
-  var tooltipContent = removeRichTextHtmlTags(this.text.tooltip);
-
-  function removeRichTextHtmlTags(str) {
-    var _str = String(str);
-    return _str.replace(/<\/?[^>]+(>|$)/g, "");
-  }
-
-  return {
-    labelPath: labelPath,
-    shortDescriptionPath: shortDescriptionPath,
-    longDescriptionPath: longDescriptionPath,
-    questionMarkPath: questionMarkPath,
-    errorMessagePath: errorMessagePath,
-    clientlibs: clientlibsArr,
-    tooltipContent: tooltipContent
-  }
+use(["../../../base/v1/base/base.js"], function (Base) {
```

Comment:
If someone customizes the component, they would have to change this. We can't use absolute URL's since this can be part of /apps or /libs